### ISSUE TITLE
ExtractRefs.who_iris (and others, probably) produce references with titles that are actually long sentences/paragraphs

### ISSUE THREAD

**Issue by hblanks** [wellcometrust/policytool#237]

**Comment by dd207** [comment]
Am I right in thinking that we've done some work to address this @ivyleavedtoadflax and therefore this can be closed?

**Comment by ivyleavedtoadflax** [comment]
Good question. Yes this will get superseded by the deep_reference_parser once https://github.com/wellcometrust/reach/pull/317 gets merged back in to master, which is blocked by infrastructure issues at present.

**Comment by ivyleavedtoadflax** [comment]
So the issue still exists, but it's probably not worth fixing...
